db/python/layers/seqr.py

```diff
@@ -0,0 +1,525 @@
+# pylint: disable=unnecessary-lambda-assignment,too-many-locals
+
+import os
+import re
+import asyncio
+from collections import defaultdict
+from datetime import datetime
+from typing import Iterable, Iterator, TypeVar
+
+import aiohttp
+from cloudpathlib import AnyPath
+from cpg_utils.cloud import get_google_identity_token
+
+from api.settings import SEQR_URL, SEQR_AUDIENCE, SEQR_MAP_LOCATION
+from db.python.connect import Connection
+from db.python.layers.analysis import AnalysisLayer
+from db.python.layers.base import BaseLayer
+from db.python.layers.family import FamilyLayer
+from db.python.layers.participant import ParticipantLayer
+from db.python.layers.sequence import SampleSequenceLayer
+from db.python.tables.project import ProjectPermissionsTable
+from db.python.utils import ProjectId
+from models.enums import SequenceType, AnalysisStatus, AnalysisType
+from models.models.sample import sample_id_format
+
+# literally the most temporary thing ever, but for complete
+# automation need to have sample inclusion / exclusion
+SAMPLES_TO_IGNORE = {22735, 22739}
```

Comment:
Could we use Metamist's concept of "disabled" samples for now?

Reply:
We don't really have it, it's really poor and inconsistently implemented. Would be good to overhaul as part of upcoming refactors :)